Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Arakkoa Faiths and Names

## Changelog:
- Added 3 arakkoa faiths in their own group, Avianism:
  - Anzuism
  - Rukhmarism
  - Sethekk
- Added name sets for the Arakkoa:
<details>
<summary>Male</summary>

```
Akizzi Anjuk Bra'tkireerr Brakma Bralksaryiks Bralkseth Braylthrynis Brith Brokk Cha'kym Chaks
Chal Chazrir Che'kiss Chigikks Chikron Cho'kelyaik Cho'kkaa Chod Cholnutiz Chorr Desh Dezyr
Do'skerr Dokmi Duah Duatkyx E'skox Ekkizal Ekktyrr Ekroas Elksuruuas Engeylyos Er'thevok Esrysh
Essku Evrorruath Fad Falnukkox Fazee Fekrylyekk Fesfekkeeth Fish Fuuagaak Graasfyss Graassku
Gresh Gresskotix Grezox Gribiz Grikluk Grish Gruatkykk Gruuahaz I'rroh I'saves I'zo Ikzux
Ilazzar Isfetakks Issakks Issuvesh Ivraarrif Izzezikk Jaks Jaynyryoa Jec Jekks Jiktyves Jisskar
Jongoozearr Jot Kakks Kakra Kektorras Kerykks Kik Kizzyrra Kokrozzex Koktuta Kosfor Krasskonef
Kre'skeyroax Krikks Krivrevir Krohoc Krosoz Krostesh Krowk Kuuakzyf Laikk Lalkorr Lekmeryos
Lerran Libeetess Litkinix Lokks Lokleros Makk Malketaks Marath Marshi Mebivir Meth Mex Meze Mok
Morduf Mos O'tooryac Oklitir Okres Olurr Ongyvaf Onjuss Ordis Ordok Orruneth Pa'tanir Pam
Pe'syth Pershynaath Pess Pih Pokk Por Pordu Ranjevikks Rardirress Rawk Re'rrykke Reeth Rehis
Reth Ribykks Ril Rolnaryak Ross Sakk Sayk Sazzuss Selksyrr Sikutoth Skak Skarsheetakks Skikks
Skokks Skovrum Skrakk Skrazrokketh Skrekktyr Skrelksiks Skrerdex Skrorr Sra'skesh Sragas
Sralkezoks Srash Sratkez Sre'rynaad Sreshkuur Sri'sku Srikkiss Srir Srix Srolkush Srotketim
Sruas Ssatkok Ssedorr Sseeth Sselksuux Ssessukkuuaks Ssir Sso'skukk Ssonjulyiks Ssosh Tah Te'ruu
Tektara Teth Tetkulyoax Thayrri Thi'zurikks Thik Thothytin Tidoavekk Tihym Tirdass Varshok
Velthreerr Vesh Vok Vozas Zaashnes Zah Zalkur Zar Zar'thozzakk Zasfekk Zavrysh Zinjezzash Zith
Zo'keryor Zo'skash Zoks Zollik Zoth Zuad
```

</details>

<details>
<summary>Female</summary>

```
A'kkex Adax Aekhex Aekhissex Alrax Alsrax Alsresex Alzres Arakat Avax Ekith Ekligaf Eklirri
Ektisi Elissy Elksec Eogiky Eolksax Eolsex Erzessax Faklex Falrirrath Falzassyx Felis Firze
Fokix Fra'kkix Frae'kkassat Frelthyx Frokkax Froklerrif Froltex Frulzyx Fugy I'kix Ikix Ilsrit
Iltix Irkyx Irnix Itharrex Ithyx Jalen Jelri Jorasex Jukka Julsezex Ka'rres Kae'kax Kilsrax
Kraelzax Kraheta Kralksex Kregix Kreokhix Kroklex Krorkix Krovim Kru'kix Kulthezzix Kurlax
Kuvane Lalsrekax Lashe Lelzra Lilksex Lirkrez Lo'rrisa Lornix Lulsezzath Madekkix Mae'kkit
Maretyx Mekhati Meltyt Merken Mo'kyta Modakyx Molthex Mukkix Mulzrirrex Murlix Nadave Naeltanym
Nakka Negex Neoklex Nesheze Ni'kanix Ni'kyx Ni'rrex Niktin Nilzrix Nolkset Nu'kix Nulksi
O'karrec O'rarrix Oda Okhax Okhyt Okla Oklyx Olksinix Orax Orkax Orla Orzizam Othys Rarlynix
Rernyx Rilsax Ro'ka Rolra Rulrix Sagyth Salksix Salthix Selsran Shaeren Shaktix She'kkazzeth
Shekhava Shelax Shelsex Shelthakaf Shelzem Shihax Shishetex Sholsam Shralthyx Shrarlissax
Shrirla Shruklix Shrushex Shuhessa Shuktyzzyx Silksyx Ska'rrarrax Skarzi Skekam Skeorec Skerlex
Skihem Skirkraf Skivex Skukex Skulzrex Solzex Sorlac Sornyzi Sra'kkex Sraklyx Srarkrasa Sre'kax
Srilriryh Srilrix Srishex Srogyx Srolksaga Srulksax Sseo'rrizzen Sseran Ssi'ra Ssigix Ssirkax
Ssirkry Ssohym Ssorkres Ssovekkyx Ssulksit Ssurlirri Su'rex Su'rrix Sukhis Sulzryx Tharzat
Thi'kix Thilryssa Thogex Thohax Thokhyx Thoktysyx Tholsex Thu'ketex Thu'ryx Thulzan Ukex
Ukkarrex Ukkix Uktaf Ulrax Ulthath Ultyx Vahassex Valzregat Ve'rrax Vekkix Vikka Vo'rrex
Vurzerrix Za'kki Zahyx Zalsryth Zalzekkax Ze'rerrax Zeornasyx Zi'rresin Zihyx Zilzith
```

</details>


# How to test:
@Warcraft-GoA-Development-Team/testers, open the **_Religion Tab_**,  click **_Other Faiths_**, check **_Only Your Faiths_**, find these new faiths.
Tell your opinion about tenets, doctrines, descriptions.

@Warcraft-GoA-Development-Team/english-proofreaders, [Filtered Only Loc Files](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/401/files?file-filters%5B%5D=.yml).
Feel free to change the texts as you want, but keep the same length.
